snap-build/amd64/run_qemu.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -x
+
+WORKDIR=${WORKDIR:-.}
+source ${WORKDIR}/lib.sh
+
+ip="${1}"
+port="${2}"
+arch_dir="${WORKDIR}/amd64"
```

Comment:
There are lots of variables here that can be shared between the different scripts - see comment above about `config.sh`.